pydis_site/apps/content/resources/guides/python-guides/windows/putting-python-on-path.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Adding Python to the Path
+description: How to make sure Python is properly on the Windows Path environment variable
+icon: fab fa-windows
+toc: 2
+---
+
+If you're on Windows and know you have [Python installed from python.org](https://www.python.org/downloads/) ([our
+recommended way](../installing-and-using-python)) but you're still getting errors like
+
+```text
+pip : The term 'pip' is not recognized as the name of a cmdlet, function, script file, or operable program.
+```
+
+when trying to run a [`pip`](https://pip.pypa.io/en/stable/) or [`pyinstaller`](https://pypi.org/project/pyinstaller/)
+command, or running `python` [unexpectedly opens up the Microsoft
+Store](../microsoft-store#confusing-app-execution-alias-behaviour), the issue is likely that your Windows Path
+environment variable is improperly configured for Python. This just means Windows doesn't know how to find your Python
+installation.
+
+The Path (or PATH) environment variable is what Windows uses to locate executables needed on the command line. Whenever
+you type a command name in your terminal, like `python` or `pip`, the terminal looks for a program of the same name in
+the folders listed in the Path, so the command can be run.
+
+The [python.org Python installer](https://www.python.org/downloads/) provides an option to add Python to your Path
+automatically, although it is not checked by default so many people miss it:
+
+[![Add Python to path installer checkbox.](/static/images/content/python-on-windows/installer_path_checkbox.png)](/static/images/content/python-on-windows/installer_path_checkbox.png)
+
+But don't worry, you can have the installer add it after the fact, or add it yourself! Continue on just below to the
+[safe method](#safe-method-let-the-installer-add-python-to-path) to have the installer put Python on Path for you. Or,
+if you want to learn a handy Windows programming skill, skip to the [advanced
+method](#advanced-method-manually-edit-the-path) that explains how to edit the Path manually.
+
+## Safe method: Let the installer add Python to Path
+
+The easiest and safest way to add Python to Path is by rerunning the installer. For this you will need the same
+installer you used to install Python, which you can download again if needed from
+[python.org](https://www.python.org/downloads/windows/), or it may still be in your Downloads folder (it's a file named
+something like `python-3.11.0-amd64.exe`).
```

Comment:
Can you just access it through "apps and features -> python -> modify" in settings.

Reply:
You can but, as I discovered, you'll still need the installer around to finally make the changes. This is mentioned in parentheses just before the "Verifying your changes" section.

I figured, since they need it anyway, better to just run the installer than look for python on Apps & features.

Reply:
Makes sense